### PR TITLE
[Doc]Update a solution in loading faq

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Review by luohaha — DISMISSED** [review]

File: docs/zh/faq/loading/Loading_faq.md [hunk 1]
```diff
@@ -10,13 +10,17 @@ displayed_sidebar: "Chinese"
 
 - 增大单次导入的数据量，降低导入频率。
 
-- 在 BE 的配置文件 **be.conf** 中修改以下配置，通过调整合并策略实现加快合并的目的：
+- 修改 BE 的配置文件 **be.conf** 的相关配置，通过调整合并策略实现加快合并的目的：
+  
+  - 对于明细表，聚合表和更新，可以适当调大 `cumulative_compaction_num_threads_per_disk`、`base_compaction_num_threads_per_disk` 和 `cumulative_compaction_check_interval_seconds` 的值。例如：
```
Comment (luohaha) [inline docs/zh/faq/loading/Loading_faq.md:15]: 这里"聚合表和更新" -> 聚合表和更新表 ？

File: docs/en/faq/loading/Loading_faq.md [hunk 2]
```diff
@@ -11,13 +11,17 @@ You were running load jobs too frequently, and the data was not compacted in a t
 - Increase the amount of data loaded in each individual job, thereby reducing loading frequency.
 
 - Modify the configuration items in the BE configuration file **be.conf** of each BE as follows, thereby accelerating data compactions:
+   
+  - For Duplicate key tables, Aggregate Key tables, and Unique Key tables, you can appropriately increase the values of `cumulative_compaction_num_threads_per_disk`, `base_compaction_num_threads_per_disk`, and `cumulative_compaction_check_interval_seconds`. For example:
```
Comment (amber-create) [inline docs/en/faq/loading/Loading_faq.md:15]: image
-> Aggregate tables

File: docs/zh/faq/loading/Loading_faq.md [hunk 3]
```diff
@@ -10,13 +10,17 @@ displayed_sidebar: "Chinese"
 
 - 增大单次导入的数据量，降低导入频率。
 
-- 在 BE 的配置文件 **be.conf** 中修改以下配置，通过调整合并策略实现加快合并的目的：
+- 修改 BE 的配置文件 **be.conf** 的相关配置，通过调整合并策略实现加快合并的目的：
```
Comment (evelynzhaojie) [inline docs/zh/faq/loading/Loading_faq.md:13]: BE 的配置文件 --- BE 配置文件，合并策略 -- Compaction 策略

File: docs/en/faq/loading/Loading_faq.md [hunk 4]
```diff
@@ -11,13 +11,17 @@ You were running load jobs too frequently, and the data was not compacted in a t
...
 
     ```Plain
     cumulative_compaction_num_threads_per_disk = 4
     base_compaction_num_threads_per_disk = 2
     cumulative_compaction_check_interval_seconds = 2
     ```
 
+  - For Primary key tables, you can appropriately increase the value of `update_compaction_num_threads_per_disk` and decrease the value of `update_compaction_per_tablet_min_interval_seconds`.
```
Comment (amber-create) [inline docs/en/faq/loading/Loading_faq.md:23]: image
-> Primary Key tables

File: docs/zh/faq/loading/Loading_faq.md (hunk 3 above)
Comment (amber-create) [inline docs/zh/faq/loading/Loading_faq.md:13]: image
-> 修改 BE 配置文件 **be.conf**   或者   修改 BE 配置文件 **be.conf** 中相关参数的配置

File: docs/zh/faq/loading/Loading_faq.md [hunk 5]
```diff
@@ -10,13 +10,17 @@ displayed_sidebar: "Chinese"
...
+  
+  - 对于明细表，聚合表和更新表，可以适当调大 `cumulative_compaction_num_threads_per_disk`、`base_compaction_num_threads_per_disk` 和 `cumulative_compaction_check_interval_seconds` 的值。例如：
```
Comment (amber-create) [inline docs/zh/faq/loading/Loading_faq.md:15]: image
顿号

File: docs/zh/faq/loading/Loading_faq.md [hunk 6]
```diff
@@ -10,13 +10,17 @@ displayed_sidebar: "Chinese"
...
+      base_compaction_num_threads_per_disk = 2
+      cumulative_compaction_check_interval_seconds = 2
+      ```
 
-    ```Plain
-    cumulative_compaction_num_threads_per_disk = 4
-    base_compaction_num_threads_per_disk = 2
-    cumulative_compaction_check_interval_seconds = 2
-    ```
+  - 对于主键表：可以适当调大 `update_compaction_num_threads_per_disk` 的值。适当调小 `update_compaction_per_tablet_min_interval_seconds` 的值。
```
Comment (amber-create) [inline docs/zh/faq/loading/Loading_faq.md:23]: image
逗号

File: docs/zh/faq/loading/Loading_faq.md (hunk 5 above)
Comment (evelynzhaojie) [inline docs/zh/faq/loading/Loading_faq.md:15]: 可以适当调大 BE 参数 xxx

File: docs/zh/faq/loading/Loading_faq.md (hunk 6 above)
Comment (evelynzhaojie) [inline docs/zh/faq/loading/Loading_faq.md:23]: 可以适当调大 BE 参数 xxx

File: docs/en/faq/loading/Loading_faq.md (hunk 2 above)
Comment (evelynzhaojie) [inline docs/en/faq/loading/Loading_faq.md:15]: Aggregate Key tables --- Aggregate tables

File: docs/en/faq/loading/Loading_faq.md (hunk 4 above)
Comment (evelynzhaojie) [inline docs/en/faq/loading/Loading_faq.md:23]: value of BE parameter

Comment (evelynzhaojie) [inline docs/en/faq/loading/Loading_faq.md:23]: Primary Key

File: docs/en/faq/loading/Loading_faq.md (hunk 2 above)
Comment (evelynzhaojie) [reply docs/en/faq/loading/Loading_faq.md:15]: Duplicate Key

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**